lib/generator/code_generator.dart

```diff
@@ -10,7 +10,7 @@ import 'package:graphql_generator/generator/objects.dart';
 import 'package:graphql_generator/generator/union.dart';
 
 class GraphQLCodeGenerators {
-  generate(
+  String generate(
```

Comment:
What should this function be called? I see it's returning a mutationObjectType but I don't think `GraphQLCodeGenerators().generate(...)` would be very clear about what the method is doing.

Reply:
It would return the complete dart code that we generated.